src/core/main.c

```diff
@@ -698,6 +698,19 @@ static int parse_config_file(void) {
 
         config_parse_many(fn, conf_dirs_nulstr, "Manager\0", config_item_table_lookup, items, false, NULL);
 
+        _cleanup_free_ char *rvalue = NULL;
+        if (get_proc_cmdline_key("systemd.default_timeout_start_sec=", &rvalue)) {
+                usec_t jobtimeout;
+                int r;
+
+                r = parse_time(rvalue, &jobtimeout, 1);
+                if (r < 0) {
+                        log_warning("Failed to set job timeout to %ss", rvalue);
```

Comment:
please use log_warning_errno() and pass "r" as first argument. Also, this message is misleading as this is purely a parsing error. On top of that, `%ss` is simply wrong. The point of `parse_time()` is after all that it can parse strings like "6h20min" in a sensible way. Suffixing a string like that with "s" certainly makes no sense.

Also, the log message should really say, that the parse error is ignored. Something like this:

```c
…
log_warning_errno(r, "Failed to parse default start timeout: %s", rvalue);
…
```